Dockerfile

```diff
@@ -1,46 +1,86 @@
-FROM python:3.6.4-alpine3.7
-MAINTAINER Shane Frasier <jeremy.frasier@trio.dhs.gov>
 ###
-# Install shadow, so we have adduser and addgroup.  We can remove the
-# package once we're finished with it.
+# Install everything we need
+###
+FROM python:3-alpine as install
+LABEL maintainer="jeremy.frasier@trio.dhs.gov"
+LABEL organization="CISA Cyber Assessments"
+LABEL url="https://github.com/cisagov/saver"
+
+ENV HOME=/home/saver
+ENV USER=saver
+
+###
+# Dependencies
 #
 # We need redis so we can use redis-cli to communicate with redis.  I
 # also reinstall wget with openssl, since otherwise wget does not seem
 # to know how to HTTPS.
 ###
-RUN apk --no-cache add wget openssl redis shadow
+ENV DEPS \
+    bash \
+    openssl \
+    redis \
+    wget
+RUN apk --no-cache --quiet add $DEPS
```

Comment:
It might be beneficial to add to the dependencies comment to explain `--no-cache`. I had to look into apk's flags to see why we weren't bothering with a `--update`.

Reply:
Great suggestion!  Addressed in 6781a3e.